letsencrypt/tasks/main.yml

```diff
@@ -85,6 +90,28 @@
   when: letsencrypt_run|bool == true and letsencrypt_certbot_plugin == "webroot"
   tags: ['letsencrypt', 'letsencrypt:run']
 
+- name: Renew the webroot certificate if needed
+  # certbot is smart enough to renew only when needed
+  # This task is useful till we solve the LetsEncrypt cronjob issue
+  command: >
+    {{ letsencrypt_command }} renew --cert-name {{ item.domains[0] }}
+  with_items: "{{ letsencrypt_certs }}"
+  become: True
+  become_user: "{{ letsencrypt_webuser }}"
+  when:
+    - letsencrypt_run|bool == true
+    - letsencrypt_certbot_plugin == "webroot"
+  tags: ['letsencrypt', 'letsencrypt:run', 'letsencrypt:renew']
+
+- name: Reload nginx
+  # Reload nginx one more time just in case a certificate was renewed
+  service: name=nginx state=reloaded
+  when:
+    - letsencrypt_run|bool == true
+    - letsencrypt_certbot_plugin == "webroot"
+    - letsencrypt_webserver == "nginx"
+  tags: ['letsencrypt', 'letsencrypt:configuration', 'letsencrypt:renew']
```

Comment:
Let's maybe take out the `letsencrypt:configuration` tag here. We don't need to run two reload tasks when just trying to change configuration.  Or is there a reason I'm not seeing?  It'll reload twice when running plain `letsencrypt`, too, but that seems fine--the original reload task is needed where it is, and we should always be able to renew when running the full unmodified tag.

Reply:
@bryanlandia thanks for your question. It actually makes sense to remove the `:configuration` tag.

I'd like to point out that unlike restart, a reload is very cheap and causes zero downtime.
See "Changing Configuration" section in http://nginx.org/en/docs/control.html .